CombatCrittersSharp/rest/payloads/PackPayload.cs

```diff
@@ -1,29 +1,8 @@
-using System.Net.Mime;
 
 namespace CombatCrittersSharp.rest.payloads;
 
-public record PackPayload(
-    string name,
-    string image,
-    int packid
+public record PackDetailsPayload(
+    string Name, //Name of pack
+    string Image, //Pack Image
+    int Packid //Pack Id
```

Comment:
These cannot be renamed, they are conforming to the api, the record name `PackPayload` is also reflecting api but is less vital